Mathlib/Analysis/Calculus/ContDiff/Basic.lean

```diff
@@ -274,6 +274,13 @@ theorem ContinuousLinearEquiv.iteratedFDerivWithin_comp_left (g : F ≃L[𝕜] G
       ContinuousLinearMap.compContinuousMultilinearMap_coe, EmbeddingLike.apply_eq_iff_eq]
     rw [iteratedFDerivWithin_succ_apply_left]
 
+/-- Iterated derivatives commute with left composition by continuous linear equivalences- -/
+theorem ContinuousLinearEquiv.iteratedFDeriv_comp_left {f : E → F} {x : E} (g : F ≃L[𝕜] G) {i : ℕ} :
+    iteratedFDeriv 𝕜 i (g ∘ f) x =
+      (g : F →L[𝕜] G).compContinuousMultilinearMap (iteratedFDeriv 𝕜 i f x) := by
+  simp only [← iteratedFDerivWithin_univ]
+  apply g.iteratedFDerivWithin_comp_left f uniqueDiffOn_univ trivial
```

Comment:
Can this be `exact`?

Reply:
Negative. Using "exact" produces an error message:

```
type mismatch
  iteratedFDerivWithin_comp_left g f uniqueDiffOn_univ trivial
has type
  ∀ (i : ℕ),
    iteratedFDerivWithin 𝕜 i (⇑g ∘ f) univ ?m.153113 =
      (↑g).compContinuousMultilinearMap (iteratedFDerivWithin 𝕜 i f univ ?m.153113) : Prop
but is expected to have type
  iteratedFDerivWithin 𝕜 i (⇑g ∘ f) univ x =
    (↑g).compContinuousMultilinearMap (iteratedFDerivWithin 𝕜 i f univ x) : Prop
```